src/contrib/cluster/Akka.Cluster.Tools/Client/ClusterClient.cs

```diff
@@ -419,7 +607,8 @@ private bool Establishing(object message)
                 }
                 else
                 {
-                    // ok, use another instead
+                    // prune out actors that failed to be identified
+                    PruneContacts((string) actorIdentify.MessageId);
```

Comment:
Dead contact detection, remove the dead contact if it doesn't respond to `Identify`